CHANGELOG

```diff
@@ -1,30 +1,74 @@
 cocos2d-x-3.10 December ? 2015
-[NEW]           Core:  Added Application::getVersion() to get the app version.
+
+[HIGHLIGHT]     UI: Rewrite Scale9Sprite and improve the scale9sprite performance and reduce memory consumption.
+[HIGHLIGHT]     UI: Change PageView to derive from ListView.
+
+[NEW]           Core: Added Application::getVersion() to get the app version.
 [NEW]           UI: Add PageView indicator.
-[NEW]           Label: Add three Overflow type to new label, see release note for more information.
+[NEW]           UI: Label add three Overflow type to new label, see release note for more information.
 [NEW]           UI: UIText::clone supports clone the text effect.
 [NEW]           Label: Add methods to query label effect state.
+[NEW]           UI: UIRichText support the new line element.
+[NEW]           UI: Scale9Sprite fade actions with cascade opacity.
+[NEW]           UI: Pageview support adjust child size.
 
 [REFINE]        UI: RichText support new line element.
 [REFINE]        UI: Set focus to Widget when touched.
 [REFINE]        UI: Change PageView to derived from ListView.
-[REFINE]        UI: Rewrite Scale9Sprite and improve the scale9sprite performance and reduce memory consumption.
 [REFINE]        3D: Change char* to string in Terrain.
-[REFINE]        Editor: Merge Studio ActionTimeLine change back into engine.
+[REFINE]        Studio: Merge Studio ActionTimeLine change back into engine.
+[REFINE]        Studio: Merge Studio changes for compatible withe 32bit Mac System.
+[REFINE]        Studio: Merge Studio changes for lua-binding, js-binding and simulator.
 [REFINE]        Mac: Make engine compatible for 32bit Mac.
 [REFINE]        Audio: AudioEngine on Linux replace the original SimpleAudioEngine with a new version of FMOD, now AudioEngine support all platforms!
+[REFINE]        IOS: Add virtual keyword for some render related function.
+[REFINE]        UI: Fixes boring deprecated warning in HttpRequest.
+[REFINE]        Network: Fix Downloader bug on iOS & Android platform.
+[REFINE]        Studio: Fix deprecation warning in SkeletonRenderer.
+[REFINE]        JS: Add js test case for fix, improve template.
+[REFINE]        Network: Permit http access to cocos2d-x.org in test projects on iOS.
+[REFINE]        Network: Crash when removing a remotely downloaded image from texture cache in js-binding.
+[REFINE]        Win10: WinRT project update version to v3.10.
+[REFINE]        Console: Add quiet option for Cocos Toolkit.
+[REFINE]        JS: New GC model for js-binding.
+[REFINE]        Doc: Fix typos in documentation and comments.
+[REFINE]        UI: update controlButton size calculate with new Scale9Sprite logic.
+[REFINE]        Win10: Added missing _USRJSSTATIC preprocessor define for ARM builds.
+[REFINE]        JS: Added ccvector_to / ccmap_to converted to new js-binding API.
+[REFINE]        UI: Slider misprint fix.
+[REFINE]        UI: Scale9Sprite cropped sprite frames support.
```

Comment:
`[REFINE]        UI: Scale9Sprite cropped sprite frames support.` This one is not needed. The previous version of Scale9Sprite support this.